src/styles/components/buttons.css

```diff
@@ -9,55 +9,68 @@
     &[disabled] {
       @apply cursor-not-allowed;
     }
-   
-    &.text {
-      @apply text-blue;
-      @apply hover:text-blue-dark;
+
+    &.primary {
+      @apply bg-gradient-to-r from-violet to-violet-russian text-white;
 
       &[disabled] {
-        @apply text-gray-lighter;
+        @apply bg-gradient-to-r from-white-ghost to-white-ghost text-gray-magnolia;
```

Comment:
I missed that; what means `bg-gradient-to-r`? What the `r` means? You need to be more descriptive with the names.

Reply:
Those are tailwind names for specifying a gradient. Here's the doc from where I got it.
[Tailwind background images](https://tailwindcss.com/docs/background-image)